tests/integrations/linuxdeploy/test_LinuxDeploy__elf_patch.py

```diff
@@ -0,0 +1,79 @@
+from unittest.mock import MagicMock
```

Comment:
There's nowhere else for me to put this comment; but the test file name should match the method name; when you renamed the method name, the test file should come along for the ride.

Reply:
Also tracks! Resolved.